### PR TITLE
RDoc-2459_postponeBackup

### PR DESCRIPTION
**Pull request by reebhub**
Document new Backup Delay operation (Studio + Client API), fix Studio backup page

### CODE REVIEW

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown [hunk 1]
```diff
@@ -305,16 +278,61 @@ There are also some important differences:
...
 
 {PANEL/}
 
+{PANEL: Delaying Backup Operations}
+
+The execution of a running backup operation can be **delayed** for a given time period.  
+
+* A delayed backup operation is cancelled immediately.  
+* The delayed operation (be it a 
+  [one-time backup operation](../../../studio/database/tasks/backup-task#manually-creating-one-time-backups)
```
Comment (ArieSLV) [inline Documentation/5.2/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown:291]: Regrettably, the presence of the "Delay" button in the details of one-time backup execution is a bug. I've raised an issue for this bug: [RavenDB-20992](https://issues.hibernatingrhinos.com/issue/RavenDB-20992) We should not provide the ability to delay a one-time manual backup

A one-time backup can only be aborted.

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/tasks/images/backup_backup-details-popup.png
Comment (ArieSLV) [inline Documentation/5.2/Raven.Documentation.Pages/studio/database/tasks/images/backup_backup-details-popup.png:1]: I have a concern that the tooltip text `Click to Delay Backup Execution` may be hard to read.

File: Documentation/5.2/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/Maintenance/Backup/Backup.cs [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+                await docStore.Maintenance.SendAsync(new StartBackupOperation(true, result.TaskId));
+                    #endregion
+
+                #region get_backup_execution_results
+                // Pass the the ongoing backup task ID to GetPeriodicBackupStatusOperation  
+                var backupStatus = new GetPeriodicBackupStatusOperation(result.TaskId);
+                #endregion
+
+                // Delay the execution of a backup operation
+                var taskId = result.TaskId;
```
Comment (ArieSLV) [inline Documentation/5.2/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/Maintenance/Backup/Backup.cs:247]: This line can be deleted as `taskId` is never used?

Comment (reebhub) [reply Documentation/5.2/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/Maintenance/Backup/Backup.cs:247]: it's used in the following line to get the task info (seems clearer than using `result.TaskId`)
```
var taskBackupInfo = await docStore.Maintenance.SendAsync(
                        new GetOngoingTaskInfoOperation(taskId, OngoingTaskType.Backup)) as OngoingTaskBackup;
```

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/tasks/images/backup_backup-details-popup.png
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/tasks/images/backup_backup-details-popup.png:1]: enlarged (o:

**Review by ArieSLV — APPROVED** [review]

**Pull request merged**